sdk/src/main/java/com/horizen/account/api/rpc/types/SyncStatus.java

```diff
@@ -0,0 +1,24 @@
+package com.horizen.account.api.rpc.types;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import com.horizen.serialization.Views;
+
+@JsonView(Views.Default.class)
+public class SyncStatus {
+
+    public boolean syncStatus;
+    public int currentBlock;
+    public int startingBlock;
+    public int highestBlock;
```

Comment:
I updated the RPC interface modifying the `SyncStatus` object (ignoring the `syncStatus` field at _json_ level and changing the datatype of the block height fields to `BigInteger`.
I choose this way because we will have just one object external to the Actor that is used at rpc level.
The height fields will be serialized as hex after the rebase with the current `dev_evm`.